build_projects/dotnet-cli-build/dotnet-cli-build.csproj

```diff
@@ -66,7 +61,6 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
```

Comment:
Yes you should be able to remove this
`<Compile Include="**\*.cs" Exclude="bin\**;obj\**;**\*.xproj;packages\**" />`

because none of those extra exclusions are being used in this project. 

You should also be able to remove the exclude on the other Compile item.

EmbeddedResource can be removed, there's no resources in this project.